docs/download.md

```diff
@@ -6,31 +6,40 @@ sitemap:
   changefreq: "daily"
 ---
 
-## Releases
+## Rolling Release
 
-These are our current tagged releases. `Latest` is our current stable release,
-`Beta` is our current release candidate version that may still require some
-additional work.
+These releases are built on each push to our `master` branch and automatically
+built by [Cirrus CI](https://cirrus-ci.com/github/pulsar-edit/pulsar). These
+releases come with the latest fixes, updates and improvements so this is a great
+choice if you want to get the latest features as soon as possible.
+
+While we do our best to review and test each PR thoroughly before merging there
```

Comment:
Maybe add a note saying that rolling releases generally can be considered more stable than "regular" release?

Reply:
I think we have been intentionally skirting around this word as it has a lot of weight behind it that 1) we don't necessarily want to support and 2) isn't necessarily true where the rolling release has been *more* stable.